File: app/src/main/java/com/johnpetitto/orachat/data/Pagination.java
package com.johnpetitto.orachat.data;

import com.google.gson.annotations.SerializedName;

public class Pagination {
    private @SerializedName("current_page") int currentPage;
    private @SerializedName("per_page") int perPage;
    private @SerializedName("page_count") int pageCount;
    private @SerializedName("total_count") int totalCount;

    public int getCurrentPage() {
        return currentPage;
    }

    public int getPerPage() {
        return perPage;
    }

    public int getPageCount() {
        return pageCount;
    }

    public int getTotalCount() {
        return totalCount;
    }
}

File: app/src/main/java/com/johnpetitto/orachat/data/Pagination.kt
package com.johnpetitto.orachat.data

import com.google.gson.annotations.SerializedName

data class Pagination(
        @SerializedName("current_page") val currentPage: Int,
        @SerializedName("per_page") val perPage: Int,
        @SerializedName("page_count") val pageCount: Int,
        @SerializedName("total_count") val totalCount: Int
)
